config/routes.rb

```diff
@@ -9,6 +9,13 @@
     registrations: 'users/registrations'
   }
 
+  get 'documents/index'
+  get 'documents/new'
+  get 'documents/create'
+  get 'documents/destroy'
+
+  resources :documents, only: [:index, :new, :create, :destroy]
```

Comment:
them resources roi sao con phai viet them the kia
get 'documents/index'
  get 'documents/new'
  get 'documents/create'
  get 'documents/destroy'

Reply:
à cái này a Khiêm a thêm trước khi em làm mà em quên mất chưa xóa đi